pydantic/main.py

```diff
@@ -815,6 +817,24 @@ def _get_value(
         else:
             return v
 
+    @classmethod
+    def try_update_forward_refs(cls, **localns: Any) -> None:
```

Comment:
surely this takes no arguments, it also definitely needs a private name
```suggestion
    def __try_update_forward_refs__(cls) -> None:
```

Reply:
Fixed